src/lib/containers/home_page/featured-data/FeaturedDataTabs.tsx

```diff
@@ -18,60 +18,65 @@ export type FeaturedDataTabsProps = {
   sql: string
 }
 
-const FeaturedDataTabs: React.FunctionComponent<FeaturedDataTabsProps> = props => {
```

Comment:
Mostly whitespace changes, with a small alignment fix. Not sure if the regression occurred in the past or within my changes, but since the icons have been broken, the potential alignment issue wasn't visible.